.travis.yml

```diff
@@ -33,7 +33,8 @@ before_install:
 script:
   - sudo ln -sf /usr/bin/gcov-4.9 /usr/bin/gcov
   - bash -e tools/infrastructure/check_style.sh
-  - mkdir build && cd build && cmake ../ -DBUILD_TESTS=ON -DENABLE_GCOV=ON && make install
+  - echo Number of processing units available: ${PROCESSING_UNITS_COUNT}
```

Comment:
@istoilovgithub I see strange error on travis related to this line:
{:echo Number of processing units available=: command not found
Please, check & fix.